### DISCUSSION TITLE
Setting your Organisation status as Public

### DISCUSSION THREAD

**Discussion by NS007-dev** [App-Choreography/OurContributors#34] [Team Posts]
@App-Choreography/community ,
In order for this organisation badge to show up on your GitHub Profile page, please make sure your organisation status for this org is set to **Public**. Follow below to see how to do so:

1. Go to https://github.com/orgs/App-Choreography/people
2. Search your name up.
3. Follow this:
![image](https://user-images.githubusercontent.com/70807500/130362669-7eb8ac9f-41e3-489e-b88b-1fe180e18129.png)

Thank you.

**Comment by Davicho-Dev** [comment]
Done :3
